service/src/main/resources/ttf/OpenSans-Regular.ttf.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: The Open Sans Project Authors
```

Comment:
Couldn't find copyright holders. Suggesting to writing it like this.

Reply:
Looks like it should be:
```suggestion
SPDX-FileCopyrightText: Digitized data copyright  2010-2011, Google Corporation
```

Source: https://fonts.google.com/attribution

Reply:
Thanks, updated